plugins/catalog-backend-module-ldap/src/ldap/config.ts

```diff
@@ -47,6 +47,10 @@ export type LdapProviderConfig = {
   groups: GroupConfig[];
   // Schedule configuration for refresh tasks.
   schedule?: SchedulerServiceTaskScheduleDefinition;
+  // Configuration for LDAP vendor-specific attributes. If not specified, the default values will be used:
+  // - `dnAttributeName`: `entryDN`
+  // - `uuidAttributeName`: `entryUUID`
```

Comment:
This isn't correct - the defaults are different for each vendor. You probably want to document this as "overrides for the vendor-specific default attribute names" or so